css/custom-styles.css

```diff
@@ -83,3 +83,17 @@
 .col-center {
     margin: 0 auto
 }
+
+html {
+  position: relative;
+  min-height: 100%;
+}
+body {
+  margin-bottom: 75px; /*set to height of footer plus a little bit so stuff doesn't butt up right against it.*/
+}
+.footer {
+  position: absolute;
```

Comment:
There's got to be a better way (with flexbox or something), but this is good enough™.
In fact, it's fantastic, thanks for taking care of this issue